topics/imaging/tutorials/object-tracking-using-cell-profiler/tutorial.md

```diff
@@ -0,0 +1,490 @@
+>    - {% icon param-file %} *"Select the input CellProfiler pipeline"*: output of **IdentifyPrimaryObjects** {% icon tool %}
+>    - In *"new object"*:
+>        - {% icon param-repeat %} *"Insert new object"*
+>            - *"Enter the name of the object to measure"*: `Nuclei`
+>    - *"Calculate the Zernike features?"*: `No`
+>
+{: .hands_on}
+
+> ### {% icon hands_on %} Hands-on: Intensity features
+>
+> 1. {% tool [MeasureObjectIntensity](toolshed.g2.bx.psu.edu/repos/bgruening/cp_measure_object_intensity/cp_measure_object_intensity/3.1.9+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Select the input CellProfiler pipeline"*: output of **MeasureObjectSizeShape** {% icon tool %}
+>    - In *"new image"*:
+>        - {% icon param-repeat %} *"Insert new image"*
+>            - *"Enter the name of an image to measure"*: `OrigGray`
+>    - In *"new object"*:
+>        - {% icon param-repeat %} *"Insert new object"*
+>            - *"Enter the name of the objects to measure"*: `Nuclei`
+>
+{: .hands_on}
+
+
+## Track nuclei
+
+With the nuclei and the relevant features measured, we are now ready to start the tracking step!
+> ### {% icon hands_on %} Hands-on: Object tracking
+>
+> 1. {% tool [TrackObjects](toolshed.g2.bx.psu.edu/repos/bgruening/cp_track_objects/cp_track_objects/3.1.9+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Select the input CellProfiler pipeline"*: output of **MeasureObjectIntensity** {% icon tool %}
+>    - *"Enter the name of the objects to track"*: `Nuclei`
+>    - *"Choose a tracking method"*: `Overlap`
+>        - *"Maximum pixel distance to consider matches"*: `50`
+>        - *"Filter objects by lifetime?"*: `No`
+>        - *"Select display option?"*: `Color and Number`
+>        - *"Save color-coded image?"*: `Yes`
+>            - *"Name the output image"*: `TrackedNuclei`
+>
+{: .hands_on}
+
+
+## Visualize results
+
+To make sure that the tracking has gone as expected, we will have a look at the original images together with the results of the segmentation step. And we will visualize them together (in tiles) for easier comparison.
+> ### {% icon hands_on %} Hands-on: Visualize segmentation outcome
+>
+> 1. {% tool [OverlayOutlines](toolshed.g2.bx.psu.edu/repos/bgruening/cp_overlay_outlines/cp_overlay_outlines/3.1.9+galaxy0) %} with the following parameters:
+>    - {% icon param-file %} *"Select the input CellProfiler pipeline"*: output of **TrackObjects** {% icon tool %}
+>    - *"Display outlines on a blank image?"*: `No`
+>        - *"Enter the name of image on which to display outlines"*: `OrigGray`
+>        - *"Outline display mode"*: `Color`
+>            - In *"Outline"*:
+>                - {% icon param-repeat %} *"Insert Outline"*
+>                    - *"Enter the name of the objects to display"*: `Nuclei`
+>                    - *"Select outline color"*: `#ff0000`
```

Comment:
It is not possible to use this value in that parameter.

![Screenshot from 2021-03-16 14-19-29](https://user-images.githubusercontent.com/26678117/111315428-a8181680-8662-11eb-9435-3e454939295c.png)

Reply:
```suggestion
>                    - *"Select outline color"*: {% color_picker #ff000 %} (red)
```

There's also a colour box thing you can use here, I added it for circos where colour selection is common.